apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```diff
@@ -547,9 +565,26 @@ const ArticleModal: React.FC<ArticleModalProps> = ({
             setReadingProgress(progress);
         };
 
-        container?.addEventListener('scroll', handleScroll);
-        return () => container?.removeEventListener('scroll', handleScroll);
-    }, []);
+        // Only set up observers and listeners once content is loaded
+        if (!isLoading) {
```

Comment:
Could maybe return early here instead so we don't have another level of nesting:

```
if (isLoading) {
  return;
}

// ...
```